tests/docker/start.globus.run.tests.el.sh

```diff
@@ -0,0 +1,87 @@
+#! /bin/bash
+
+# install iRODS client packages built by development environment 
+/install_local_irods_packages.sh
+
+##### Create user1 in iRODS ####
+sudo -H -u irods bash -c "iadmin mkuser user1 rodsuser"
+sudo -H -u irods bash -c "iadmin moduser user1 password user1"
```

Comment:
When using `sudo` in this manner, it's usually best to set the group as well. `-g irods`, for example. This applies to other usages of `sudo` as well.

Reply:
I changed all of the references t sudo -u to use the -g option.  I also removed the above two lines as they are redundant as that user is set up when the iRODS container starts.